symphony/bdk/core/service/datafeed/datafeed_loop_v2.py

```diff
@@ -117,6 +86,8 @@ async def _retrieve_datafeed(self) -> Optional[V5Datafeed]:
         datafeeds = await self._datafeed_api.list_datafeed(session_token=session_token,
                                                            key_manager_token=key_manager_token,
                                                            tag=self._tag)
+
+        datafeeds = list(filter(lambda df: re.compile(DATAFEED_V2_ID_PATTERN).match(df.id), datafeeds))
```

Comment:
`DATAFEED_V2_ID_PATTERN` could be directly be assigned to `re.compile("^[^\\s_]+_f_[^\\s_]+$")` as compilation is costly (at least more than the matching)